source/usage-examples/watch.txt

```diff
@@ -0,0 +1,171 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+You can keep track of changes to data in MongoDB, such as changes to a
+collection, database, or deployment, by opening a **change stream**. A change
+stream allows applications to watch for changes to data and react to them.
+The change stream returns **change event** documents when changes occur.
+You can open a change stream by calling the ``watch()`` method on
+a ``MongoCollection``, ``MongoDatabase``, or ``MongoClient`` object:
+
+.. code-block:: java
+
+   ChangeStreamIterable<Document> changeStream = database.watch();
+
+The ``watch()`` method optionally takes an **aggregation pipeline**  which
+consists of an array of **stages** as the first parameter to filter and
+transform the change event output as follows:. 
+
+.. code-block:: java
+
+   List<Bson> pipeline = Arrays.asList(
+                           Aggregates.match(
+                              Filters.lt("fullDocument.runtime", 15)));
+   ChangeStreamIterable<Document> changeStream = database.watch(pipeline);
+
+The ``watch()`` method returns an instance of ``ChangeStreamIterable``, a class
+that offers several methods to access, organize, and traverse the results.
+``ChangeStreamIterable`` also inherits methods from its parent class,
+``MongoIterable`` which implements the core Java interface ``Iterable``.
+You can call ``forEach()`` on the ``ChangeStreamIterable`` to handle
```

Comment:
This seems like a good place for a new paragraph since the prior sentences describe the ``ChangeStreamIterable``and here you're describing the ``forEach()`` method.